scripts/extract-materials.js

```diff
@@ -0,0 +1,63 @@
+const fs = require('fs');
+const path = require('path');
+const prettier = require('prettier');
+const rcfile = require('rcfile');
+const postcss = require('postcss');
+const postcssDiscardComments = require('postcss-discard-comments');
+const postcssColorModFunction = require('postcss-color-mod-function');
+const postcssCustomProperties = require('postcss-custom-properties');
+const cssvariables = require('postcss-css-variables');
+const postcssImport = require('postcss-import');
+
+const relativePath = path.join(__dirname, '../materials/internals');
+const exportPath = path.join(__dirname, '../materials');
+const importPath = path.join(__dirname, '../materials/internals/index.css');
+
+const css = fs.readFileSync(importPath, 'utf8');
+
+const commentDoNotModify = `/* THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+  This file is created by the 'scripts/extract-materials.js' script. The real
+  values of the variables should be updated in '${relativePath}' */ \n
```

Comment:
This resolves to the actual path on the system, so for example `/Users/dom/repos/ui-kit/materials/internals`. This is bad because it would change every time this file gets generated on a different machine. We need to print the path relative to the root-folder, or just hardcode it.